CHANGELOG.md

```diff
@@ -12,30 +12,62 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+> #### Deprecated Configuration Parameters Removal
+> Some configuration parameters marked as deprecated in version 2.x.x are now removed in 3.0.
+
+
+### :bug: Fixed
+- Incorrect nodeId in HostSpec for RDS Multi-AZ cluster ([PR #1579](https://github.com/aws/aws-advanced-jdbc-wrapper/pull/1579)).
+- Avoid duplicates in manifest file ([PR #1607](https://github.com/aws/aws-advanced-jdbc-wrapper/pull/1607)).
+- Add expiration time for cached reader connections and refactor Blue/Green plugin ([PR #1626](https://github.com/aws/aws-advanced-jdbc-wrapper/pull/1626)).
+
+### :crab: Changed
+- Improve Javadoc ([PR #1576](https://github.com/aws/aws-advanced-jdbc-wrapper/pull/1576)) 
+- Remove mutable keys from HostSpec#hashCode and document immutability requirements ([PR #1565](https://github.com/aws/aws-advanced-jdbc-wrapper/pull/1565))
+- Deprecate `enableGreenNodeReplacement` parameter ([PR #1578](https://github.com/aws/aws-advanced-jdbc-wrapper/pull/1578))
+- Deprecate `auroraStaleDns` plugin ([PR #1590](https://github.com/aws/aws-advanced-jdbc-wrapper/pull/1590))
+- Add `initialConnection` plugin to default plugin list ([PR #1592](https://github.com/aws/aws-advanced-jdbc-wrapper/pull/1592))
+- Remove unused `StorageService` method ([PR #1597](https://github.com/aws/aws-advanced-jdbc-wrapper/pull/1597))
+- Remove deprecated `ConnectionService` classes ([PR #1598](https://github.com/aws/aws-advanced-jdbc-wrapper/pull/1598))
+- Logging improvement for RW Splitting plugin ([PR #1604](https://github.com/aws/aws-advanced-jdbc-wrapper/pull/1604))
+- Refactor dialects and host list providers to reduce code duplication ([PR #1588](https://github.com/aws/aws-advanced-jdbc-wrapper/pull/1588))
 
 ### :magic_wand: Added
-- Added support of Global Databases including and Global Database endpoint. ([PR #1573](https://github.com/aws/aws-advanced-jdbc-wrapper/pull/1573)).
+- Added support of Global Databases and Global Database endpoint. Both in-region and cross-region failovers are supported now ([PR #1573](https://github.com/aws/aws-advanced-jdbc-wrapper/pull/1573)).
+- Respect AWS region connecting with cluster reader endpoint for Global Databases ([PR #1575](https://github.com/aws/aws-advanced-jdbc-wrapper/pull/1575)).
+- Recognize RDS Proxy custom endpoints ([PR #1583](https://github.com/aws/aws-advanced-jdbc-wrapper/pull/1583)).
+- Public `PluginService` interface extended with a new method helping to identify a source of connection. Knowing whether connection is brand new or a connection obtained from a pool helps to improve connection management. Custom plugins implemented by user can now use this new method ([PR #1581](https://github.com/aws/aws-advanced-jdbc-wrapper/pull/1581)).
+- Allow to override IAM token property name ([PR #1603](https://github.com/aws/aws-advanced-jdbc-wrapper/pull/1603)).
+- Introduces the Simple Read/Write Splitting Plugin (`srw`) ([PR #1586](https://github.com/aws/aws-advanced-jdbc-wrapper/pull/1586))
```

Comment:
```
- A new Simple Read/Write Splitting Plugin (`srw`)
```
Can we link to the documentation directly for new features instead of PRs. Could we also add a blurb on how this is different than regular rw